server/cli/cli_test.go

```diff
@@ -48,6 +49,9 @@ func (c cliTest) Run(line string) {
 	var args []string
 	args = append(args, a[0])
 	args = append(args, fmt.Sprintf("-addr=%s", c.Addr))
+	// Always load server certs. Not sure if this path is a good assumption though.
+	args = append(args, fmt.Sprintf("-certs=%s",
+		path.Join(os.Getenv("GOPATH"), "src/github.com/cockroachdb/cockroach/resource/test_certs")))
```

Comment:
this will break the deployment because there we run the test binaries in an image that doesn't have Go installed. Should have the certs embedded (see `rpctest`)